packages/kbn-check-mappings-update-cli/current_fields.json

```diff
@@ -195,6 +195,15 @@
   "cases-connector-mappings": [
     "owner"
   ],
+  "cases-oracle": [
```

Comment:
merging main 😬

Reply:
We recently merged a PR to update sorting of keys in this snapshot, probably what led the conflict. Ordering looks good to me though, but yeah this is a noop ito actual changes.